### PR TITLE
#74 - 01 Styles/05 Import materialize 

### PR DESCRIPTION
**Pull request by fjcalzado**
Done! A new sample numbered as 05 as been added to 01 Styles to import materialize framework.

### CODE REVIEW

File: 01 Styles/05 Import Materialize/README.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+  module.exports = {
+    entry: {
+      app: './students.js',
+      appStyles: [
+        './mystyles.css',
+      ],
+      vendor: [
+        'jquery',
+  +     './node_modules/materialize-css/dist/js/materialize.js',
```
Comment (nasdan) [inline 01 Styles/05 Import Materialize/README.md]: Did you try add materialize vendor like:

```
vendor: [
  'jquery',
  'materialize-css'
]
```

File: 01 Styles/05 Import Materialize/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  devtool: 'inline-source-map',
+  devServer: {
+    port: 8080,
+  },
+  plugins: [
+    //Generate index.html in /dist => https://github.com/ampedandwired/html-webpack-plugin
+    new HtmlWebpackPlugin({
+      filename: 'index.html', //Name of file in ./dist/
+      template: 'index.html', //Name of template in ./src
+			hash: true,
```
Comment (nasdan) [inline 01 Styles/05 Import Materialize/webpack.config.js]: Replace tabs by spaces

**Comment by nasdan** [comment]
Great work @fjcalzado, I only did two comments.

One of them is related with this issue #62

File: 01 Styles/05 Import Materialize/webpack.config.js (hunk 2 above)
Comment (fjcalzado) [reply 01 Styles/05 Import Materialize/webpack.config.js]: Weird thing, my VSCode is set up to replace tabs by spaces ... :S. I'll check it out at home.

File: 01 Styles/05 Import Materialize/README.md (hunk 1 above)
Comment (fjcalzado) [reply 01 Styles/05 Import Materialize/README.md]: No I didn't, but you are right, the shortcut should work. Just one question regarding this, how does webpack know which JS must target? It must be defined somewhere in Materialize right? Cause Materialize comes with 2 JS alternatives, minified and no-minified.

Comment (nasdan) [reply 01 Styles/05 Import Materialize/README.md]: In [package.json](https://github.com/Dogfalo/materialize/blob/master/package.json#L7), you can decide which .js is your main file which it's going to run when you add it to app or for example which webpack goes to take when you add to vendor section.

Comment (nasdan) [reply 01 Styles/05 Import Materialize/README.md]: Anyway, you can left like this. It only was a comment to know about it

**Comment by fjcalzado** [comment]
Done!

**Review by nasdan — APPROVED** [review]

**Pull request merged**